phpstan.neon

```diff
@@ -0,0 +1,14 @@
+parameters:
+    level: 8
+    paths:
+        - src
+    bootstrapFiles:
+        - tests/bootstrap.php
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
+    ignoreErrors:
+        - '#.+Generated\\Shared\\Transfer.+#'
```

Comment:
Line could be removed as well, because Transfers are getting generated with standalone testing
```suggestion
        - '#.+Generated\\Shared\\Transfer.+#'
```